npm/pkg/controlplane/controllers/v2/namespaceController.go

```diff
@@ -221,7 +221,7 @@ func (nsc *NamespaceController) processNextWorkItem() bool {
 }
 
 // syncNamespace compares the actual state with the desired, and attempts to converge the two.
-func (nsc *NamespaceController) syncNamespace(nsKey string) error {
+func (nsc *NamespaceController) syncNamespace(nsKey string) (err error) {
```

Comment:
need to return err everywhere too (sometimes return nil)

Reply:
Implementing the fix here: https://github.com/Azure/azure-container-networking/pull/1564